Support for Linear (USDT) on the updated typescript version. (#75)
tiagosiebler/bybit-api: README.md

## Issues & Discussion
- Issues? Check the [issues tab](https://github.com/tiagosiebler/bybit-api/issues).

> - RestClient not defined? With the recent addition of LinearClient, we renamed RestClient to InverseClient. Change all old references.
> - Discuss & collaborate with other node devs? Join our [Node.js Algo Traders](https://t.me/nodetraders) engineering community on telegram.

Reviewer: Silly request:
```suggestion
- Discuss & collaborate with other node devs? Join our [Node.js Algo Traders](https://t.me/nodetraders) engineering community on telegram.
- `'bybit-api' has no exported member 'RestClient'`: use `InverseClient` instead of `RestClient`
```
